src/main/java/com/mongodb/hibernate/jdbc/MongoDatabaseMetaData.java

```diff
@@ -18,16 +18,15 @@
 
 import static com.mongodb.hibernate.internal.MongoAssertions.assertTrue;
 import static com.mongodb.hibernate.internal.MongoAssertions.fail;
+import static com.mongodb.hibernate.internal.MongoConstants.MONGO_DBMS_NAME;
+import static com.mongodb.hibernate.internal.MongoConstants.MONGO_JDBC_DRIVER_NAME;
 
 import java.sql.Connection;
 import java.sql.DatabaseMetaData;
 import java.sql.ResultSet;
 
 final class MongoDatabaseMetaData implements DatabaseMetaDataAdapter {
 
-    public static final String MONGO_DATABASE_PRODUCT_NAME = "MongoDB";
-    public static final String MONGO_JDBC_DRIVER_NAME = "MongoDB Java Driver JDBC Adapter";
```

Comment:
I needed to use `MONGO_DATABASE_PRODUCT_NAME` in `MongoDialect`, but did not want to make `MongoDatabaseMetaData` public, so I moved the constants to `MongoConstants`, which seems appropriate.